### ISSUE TITLE
Order of class names

### ISSUE THREAD

**Issue by gireeshkbogu** [qubvel/segmentation_models.pytorch#163]
I see the dataloader has the following order of classes. It is not alphabetical. 
My question is, how could you choose the order of your own mask classes?

```
    CLASSES = ['sky', 'building', 'pole', 'road', 'pavement', 
               'tree', 'signsymbol', 'fence', 'car', 
               'pedestrian', 'bicyclist', 'unlabelled']
```
